its/projects/DuplicateAnalyzerReferences/DuplicateAnalyzer/DuplicateAnalyzer.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+	  <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\SourceGenerator\SourceGenerator.csproj" ReferenceOutputAssembly="false" OutputItemType="Analyzer" />
+	  <Analyzer Include="..\FakeGoogleProtobuf\bin\Debug\netstandard2.0\Google.Protobuf.dll"/>
```

Comment:
(I cannot add a comment on the DLL, so adding it here)

I'm a bit confused by having a binary checked in the `bin/Debug/.../` - normally `bin/Debug` files are excluded from source control in `.gitignore` files

- can't this be used when the test is being ran (to ensure that binary is first built, and then the result is used?
- otherwise, maybe put the DLL in a different folder (not in `bin/Debug`) and add a readme how to regenerate it